src/tests/system/tests/test_memory_cache.py

```diff
@@ -1273,7 +1275,7 @@ def test_memory_cache__invalidate_group_after_stop(client: Client, provider: Gen
 @pytest.mark.importance("high")
 @pytest.mark.cache
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_memory_cache__invalidate_groups_before_stop(client: Client, provider: GenericProvider):
+def test_memory_cache__invalidate_groups_cache_before_stop(client: Client, provider: GenericProvider):
```

Comment:
I think we should use "invalidate_group_cache" instead of "invalidate_groups_cache" to better match other test case names.

Reply:
The previous test is group_cache and then this one is groups_cache since it tests multiple groups. These tests need to be rework.